icemux.go

```diff
@@ -19,7 +19,7 @@ func NewICETCPMux(logger logging.LeveledLogger, listener net.Listener, readBuffe
 
 // NewICEUDPMux creates a new instance of ice.UDPMuxDefault. It allows many PeerConnections to be served
 // by a single UDP Port.
-func NewICEUDPMux(logger logging.LeveledLogger, udpConn net.PacketConn) ice.UDPMux {
+func NewICEUDPMux(logger logging.LeveledLogger, udpConn ice.UDPMuxConn) ice.UDPMux {
```

Comment:
Can we add a comment indicating `net.UDPConn` satisfies this interface? This would not impact things for most users that are using a UDPConn.